archivers/gnutar/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                gnutar
 version             1.34
-revision            1
+revision            2
```

Comment:
One tip for next time: When a port's ultimate installed layout/contents changes, you'll also need to increase the revision.

Took care of it for you, but just a heads-up for next time. An no worries, there's a lot of small details like this to learn!

Reply:
@mascguy Thanks! Would it make sense to add this point to default PR comment template? Or are there too many such nitpicks to add them all?

Reply:
> @mascguy Thanks! Would it make sense to add this point to default PR comment template? Or are there too many such nitpicks to add them all?

I definitely like the idea! There are a lot of details though, so it might be preferable to include a link to one or more of our wiki pages. Our [Committers Tips and Tricks](https://trac.macports.org/wiki/CommittersTipsAndTricks) might be a good one, and perhaps the overall [MacPorts Guide](https://guide.macports.org/).

Your thoughts?

Reply:
@mascguy Well, I was thinking more about extending the present checklist, for people doing small changes.  If you do a large change, then perhaps you'll read the docs anyway (still, a link to them would be useful).